test/lsp/lsp_hover.ml

```diff
@@ -71,6 +66,12 @@ let%expect_test "hover-copy" =
     ----          ^^^^^^^^^^^^^^^
        7          PROCEDURE DIVISION.
        8             DISPLAY FIELD
+    ```cobol
+    FIELD
+    ```
+    USAGE DISPLAY
+    ALPHANUMERIC(1)
+    Additional pre-processing information:
     ```cobol
            01 FIELD PIC X.
     ```
```

Comment:
This is the result when combining both data item info and pp info. Not sure it's good to have both, as there is not necessarily a single underlying variable to display (in case there is 2, it display the last one)